### PR TITLE
增加注释，单例方法未采用private 方法

### PR DESCRIPTION
**Pull request by lipeishen**

### CODE REVIEW

**Comment by lipeishen** [comment]
通过

File: brpc-java-core/src/main/java/com/baidu/brpc/compress/CompressManager.java [hunk 1]
```diff
@@ -37,6 +38,7 @@ public static CompressManager getInstance() {
         return instance;
     }
 
+    // why not  use private methodg
```
Comment (wenweihu86) [inline brpc-java-core/src/main/java/com/baidu/brpc/compress/CompressManager.java]: private is also ok

**Comment by jovany-wang** [comment]
Could you plz fix this? @lipeishen

**Comment by jovany-wang** [comment]
The related issue: https://github.com/baidu/brpc-java/issues/38

**Comment by jovany-wang** [comment]
@wenweihu86 I'd like to merge this PR as a linting-fix firstly.

**Comment by wenweihu86** [comment]
@jovany-wang @lipeishen ok

**Pull request merged**